config.json

```diff
@@ -70,6 +72,24 @@
   "spentAddresses": {
     "enabled": true
   },
+  "coordinator": {
+    "address": "UDYXTZBE9GZGPM9SSQV9LTZNDLJIZMPUVVXYXFYVBLIEUHLSEWFTKZZLXYRHHWVQV9MNNX9KZC9D9UZWZ",
+    "securityLevel": 2,
+    "merkleTreeDepth": 24,
```

Comment:
@karimodm these values should be the hard-coded defaults in the code instead of in the config.json
We want to try to reduce the amount of values in the default mainnet config and let the node pick the "defaults"